op-acceptance-tests/justfile

```diff
@@ -0,0 +1,25 @@
+# Default recipe - runs the acceptance tests with default parameters
+default:
+    @just acceptance-test
+
+# Run acceptance tests against a devnet
+acceptance-test devnet="simple" gate="localnet":
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    # First run the appropriate devnet from the kurtosis-devnet directory if needed
+    # we ignore failures here because the devnet may already be running
+    (cd ../kurtosis-devnet && just {{ devnet }}-devnet) || true
+
+    # Get the absolute path to the repository root
+    REPO_ROOT=$(cd .. && pwd)
```

Comment:
nit `$(realpath ..)` ?
(as a matter of principle I don't like cd-ing around too much :))

Reply:
Agreed; updated & simplified.